### PR TITLE
Notification Feature Implementation

### PR DESCRIPTION
**Pull request by SelcukEdma**
Things to Merged:

- [ ] Created Redux Message

- [ ] Save API response to redux state

- [ ] Rendered Message to user after successful API call

- [ ] Refactor Modal and SnackBar Component

- [ ] Moved SnackBar component to parent level to component

- [ ] Clean up the reducer

### CODE REVIEW

**Review by ed-roh — APPROVED** [review]

File: src/Components/Scenes/Governance/DomainManagerTable.tsx [hunk 1]
```diff
@@ -8,23 +8,26 @@ import {
   addDomainOwnersRequestFetch
 } from '../../../State/Governance/actions';
 import { TextField } from '@material-ui/core';
+import SnackBar from '../../Presentational/Modal/SnackBar';
```
Comment (ed-roh) [inline src/Components/Scenes/Governance/DomainManagerTable.tsx:11]: So will we use this SnackBar for all our notifications? I have one created as well in the Hydration App.

File: src/Components/Scenes/Governance/SensitivityTable.tsx [hunk 2]
```diff
@@ -8,18 +8,21 @@ import {
...
-  const { sensitivity, setSelectedRemoveRowId, isLoading } = useSelector(
-    (state: any) => state.governance
-  );
+  const {
+    sensitivity,
+    setSelectedRemoveRowId,
+    isLoading,
+    message
+  } = useSelector((state: any) => state.governance);
   const titleText = `Sensitivity Levels`;
```
Comment (ed-roh) [inline src/Components/Scenes/Governance/SensitivityTable.tsx:20]: move titleText and columns to after the lines of useDispatch and before useEffect

File: src/Components/Scenes/Governance/DomainManagerTable.tsx (hunk 1 above)
Comment (SelcukEdma) [reply src/Components/Scenes/Governance/DomainManagerTable.tsx:11]: Yeah Maybe we should combine them into one.

File: src/Components/Scenes/Governance/SensitivityTable.tsx (hunk 2 above)
Comment (SelcukEdma) [reply src/Components/Scenes/Governance/SensitivityTable.tsx:20]: I will move it on my next PR

**Pull request merged**